src/math.rs

```diff
@@ -2300,13 +2364,102 @@ mod tests {
                 elevation: d(-10.)
             })
             .unwrap()
-                * enu_to_frd_pointing_north_east_and_up,
+                * enu_to_frd_pointing_north_east_and_up_rolled_upright,
             Bearing::<Frd>::build(Components {
-                azimuth: d(-1.2855),
-                elevation: d(39.9944)
+                azimuth: d(1.2855),
+                elevation: d(-39.9944)
             })
             .unwrap(),
             epsilon = 0.0001_f64.to_radians() // FRD bearing is approximate
         );
+
+        // assume forward is pointing North
+        let enu_to_frd_pointing_north_inverted =
+            unsafe { Rotation::<Enu, Frd>::from_tait_bryan_angles(d(90.), d(0.), d(0.)) };
+
+        // a bearing pointing 3° off East
```

Comment:
Maybe add to the comment here that a roll of 0º in ENU means that the plane is "upside down" compared to FRD.